engine/src/main/java/org/terasology/rendering/dag/nodes/ToneMappingNode.java

```diff
@@ -67,8 +68,8 @@ public void initialise() {
     @Override
     public void process() {
         PerformanceMonitor.startActivity("rendering/toneMapping");
-        sceneToneMapped = frameBuffersManager.getFBO(SCENE_TONE_MAPPED_FBO);
-        sceneOpaque = frameBuffersManager.getFBO("sceneOpaque");
+        sceneToneMapped = dynamicFBM.getFBO(TONE_MAPPED_URN);
+        sceneOpaque = dynamicFBM.getFBO(DefaultDynamicFBOs.ReadOnlyGBuffer.getResourceUrn());
```

Comment:
Notice how in many cases this line won't be needed once this node uses the SetViewportToSizeOf state change.